tools/extract-spec-data.js

```diff
@@ -144,6 +144,29 @@ function getSpecUrl(spec) {
 }
 
 
+/**
+ * Construct the URL of the repository from the Editor's Draft URL
+ * (only works for GitHub repositories for now)
+ *
+ * @function
+ * @param {Object} spec The spec object to parse
+ * @return {String} The URL of the repository
+ */
+function getRepositoryFromEdDraft(edDraft) {
+  edDraft = edDraft || '';
+  let tokens = edDraft.match(/^https?:\/\/([^\.]+)\.github\.io\/([^\/$]+)/i);
+  if (tokens) {
+    return 'https://github.com/' + tokens[1] + '/' + tokens[2];
+  }
+  else if (edDraft.match(/^https?:\/\/drafts\.csswg\.org\//)) {
+    return 'https://github.com/w3c/csswg-drafts';
+  }
```

Comment:
Also adding https://drafts.fxtf.org/ (https://github.com/w3c/fxtf-drafts/) and https://drafts.css-houdini.org/ (https://github.com/w3c/css-houdini-drafts/)?

Reply:
Good point! I'll add these rules. Obviously, it would be better if the W3C API returned that information directly (and it should be able to extract that info from the front metadata section)